_guidelines-v4/09-textencoding/03-namesdates/02-namesdatesspecialized/02-namesdatesgeognames.md

```diff
@@ -40,19 +40,11 @@ They may also be used to identify place name components within textual content:
 
```

Comment:
In line 9:

`Geographic names are proper noun designations for places (e.g.,. Baltimore, Maryland), natural features (e.g.,. Black Forest) or political jurisdictions (e.g.,. Quartier Latin, Paris).`

3x `e.g.,` without the additional dot after colon